docs/dev/Kibana-Reporting-Design-Proposal.md

```diff
@@ -55,13 +56,24 @@ Kibana Reports for Open Distro allows ‘Report Owner’ (engineers, including b
 1. As a user, I should be able to view, edit and delete a report definition.
 1. As a user, I should be able to enable/disable a report definition
 
+
+#### Search module
+1. As a user, I should be able to find any saved search, visualization, dashboard through a search module in Reporting Landing Page, and download it.
```

Comment:
Keeping the discovery part ("I should be able to find any ...") independent of action part ("and download it") might simplify to visualize and expand the 2 parts individually (i.e. say another action part like "and view it" or "rerun it on different date range" etc would not affect the discovery part.) -- Just a thought.